local-dev/src/components/StatusBoard.js

```diff
@@ -236,6 +238,28 @@ class StatusBoard extends Component {
     componentDidMount() {
         this.updateStatus();
         setInterval(this.updateStatus.bind(this), 60000);
+        this.setState({desktopNotify: this.getDesktopNotifyPermission()});
```

Comment:
The async dialog box presented to the user means this value is always false for the entire first visit. If the user does grant permission, the state is not updated until the page is refreshed.

One possible workaround would be to default `desktopNotify` to `false` in state in the `StatusBoard` constructor, then call `getDesktopNotifyPermission` directly here in `componentDidMount`. You could alter `getDesktopNotifyPermission` to only handle cases where the user grants permission, and instead of returning `true`, update the `desktopNotify` value in state.